### ISSUE TITLE
Unable to escape quotes in alt text for toot

### ISSUE THREAD

**Issue by PeskyPotato** [PeskyPotato/vexinglyminimal#4]
Unable to escale quotes in alt text for toot.

Currently:
![image](https://github.com/LameLemon/vexinglyminimal/assets/35942108/ede83582-b813-4d93-8aa1-cebf95da91b6)

Toot: https://hachyderm.io/@PeskyPotato/110368047320994878
